### PR TITLE
docs: improve multi-backend documentation

### PR DESCRIPTION
**Pull request by njzjz**
## Summary by CodeRabbit

- **Documentation**
	- Clarified the process of freezing a model by removing references to specific code sources.
	- Updated command syntax for calculating neighbor statistics to include TensorFlow and PyTorch flags.
	- Modified descriptions to specify model files instead of graph files.
	- Added and adjusted commands to support both TensorFlow and PyTorch for training, freezing, and testing models.
	- Introduced a tabular format for configuring parallelism settings for TensorFlow and PyTorch.

### CODE REVIEW

**Review by iProzd — APPROVED** [review]

File: doc/train/training-advanced.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ One can set other environmental variables:
 | DP_AUTO_PARALLELIZATION | 0, 1          | 0             | Enable auto parallelization for CPU operators.                                                                      |
 | DP_JIT                  | 0, 1          | 0             | Enable JIT. Note that this option may either improve or decrease the performance. Requires TensorFlow supports JIT. |
 
-## Adjust `sel` of a frozen model
+## Adjust `sel` of a frozen model {{ tensorflow_icon }}
 
 One can use `--init-frz-model` features to adjust (increase or decrease) [`sel`](../model/sel.md) of a existing model. Firstly, one needs to adjust [`sel`](./train-input.rst) in `input.json`. For example, adjust from `[46, 92]` to `[23, 46]`.
```
Comment (njzjz) [inline doc/train/training-advanced.md:175]:
```suggestion
One can use `--init-frz-model` features to adjust (increase or decrease) [`sel`](../model/sel.md) of an existing model. Firstly, one needs to adjust [`sel`](./train-input.rst) in `input.json`. For example, adjust from `[46, 92]` to `[23, 46]`.
```

**Review by wanghan-iapcm — APPROVED** [review]

File: doc/model/train-fitting-dos.md [hunk 2]
```diff
@@ -82,10 +82,26 @@ To prepare the data, we recommend shifting the DOS data by the Fermi level.
...
+dp --pt train input.json
 ```
 
+:::
+
+::::
+
 The detailed loss can be found in `lcurve.out`:
 
 ```
```
[the remark this answers is not in the document]
Comment (njzjz) [reply doc/model/train-fitting-dos.md:107]: Not bash, it is txt

File: doc/model/train-fitting-dos.md [hunk 3]
```diff
@@ -82,10 +82,26 @@ To prepare the data, we recommend shifting the DOS data by the Fermi level.
 
 The training command is the same as `ener` mode, i.e.
 
+::::{tab-set}
+
+:::{tab-item} TensorFlow {{ tensorflow_icon }}
+
+```bash
+dp --tf train input.json
+```
```
[the remark this answers is not in the document]
Comment (njzjz) [reply doc/model/train-fitting-dos.md:91]: It's bash
